transformations/julia/index.md

```diff
@@ -78,7 +78,7 @@ using CSV
 using DataFrames
 
 infile = "in/tables/source.csv"
-outfile = "out/tables/destination.csv"
+outfile = "out/tables/result.csv"

```

Comment:
panebože tak dlouho si toho nikdo nevšiml 🤦 
máš ostříží zrak :)

Reply:
to byla spis nahoda :-)